src/e2e-test/regression-test/data/hl7v2/ADT_A01.hbs/ADT_A01.hbs-ADT01-23.hl7.json

```diff
@@ -173,20 +173,6 @@
                 "url": "Encounter/14693cea-02b9-3743-9010-fa549107ea68"
             }
         },
-        {
-            "fullUrl": "urn:uuid:4cfe8d6d-3fc8-3e41-b921-f204be18db31",
-            "resource": {
-                "resourceType": "Procedure",
-                "id": "4cfe8d6d-3fc8-3e41-b921-f204be18db31",
-                "subject": {
-                    "reference": "Patient/64bac34e-e611-3549-848b-89416176aa0b"
-                }
-            },
-            "request": {
-                "method": "POST",
-                "url": "Procedure/4cfe8d6d-3fc8-3e41-b921-f204be18db31"
-            }
-        },
```

Comment:
Does the regression test work with both generateUUID and generateUUIDv2?

Reply:
Not exactly. The function of "generateUUIDV2" and that of "generateUUID" are different, we can't expect they will produce the same UUID. The plan is that if we decide to write(or automatically generate) new templates, we will use "generateUUIDV2" instead of "generateUUID", and hence the corresponding ground truths will be going to change.